src/userbase-server/admin.js

```diff
@@ -246,15 +246,15 @@ const _getSizeAllowed = (admin) => {
 
 const _buildAdminResult = (admin) => {
   return {
-    email: admin['email'],
-    fullName: admin['full-name'],
-    paymentStatus: admin['stripe-saas-subscription-status'],
-    cancelSaasSubscriptionAt: admin['stripe-cancel-saas-subscription-at'],
-    connectedToStripe: admin['stripe-account-id'] ? true : false,
-    storageSubscriptionStatus: admin['stripe-storage-subscription-status'],
-    cancelStorageSubscriptionAt: admin['stripe-cancel-storage-subscription-at'],
-    altPaymentStatus: admin['alt-saas-subscription-status'],
-    size: admin['size'],
+    email: admin["email"],
+    fullName: admin["full-name"],
+    paymentStatus: "active",
```

Comment:
Assuming good faith, you did this for your local setup? Can't have this for production